### ISSUE TITLE
Explore and About menus cannot be activated from the keyboard

### ISSUE THREAD

**Issue by arteymix** [PavlidisLab/GemBrow#82]

**Comment by ppavlidis** [comment]
Gemma 2 disposition: Won't fix (UIB handoff)
